website/content/docs/commands/pki/health-check.mdx

```diff
@@ -0,0 +1,326 @@
+---
+layout: docs
+page_title: pki health-check - Command
+description: |-
+  The "pki health-check" command verifies the health of the given PKI secrets
+  engine mount against an optional configuration.
+---
+
+# pki health-check
+
+The `pki health-check` command verifies the health of the given PKI secrets
+engine mount against an optional configuration.
+
+This runs with the permissions of the given token, reading various APIs from
+the mount and `/sys` against the given Vault server
+
+Mounts need to be specified with any namespaces prefixed in the path, e.g.,
```

Comment:
So we don't support the `-namespace` or environment variable to specify the namespace?

Reply:
@stevendpclark Correct, I don't know how `-namespace` interacts with this, given that we need to access non-namespaced paths (like `sys`)... In general, the same problem holds for Kit's stuff, as the various roots and intermediates might be on different namespaces, so using namespace-in-path is at least consistent within the PKI stuff.